### ISSUE TITLE
on add tag enable scroll

### ISSUE THREAD

**Issue by rbalet** [mhutchie/vscode-git-graph#397]
**Describe the improvement that you'd like**
When I'm adding a tag, I would like to be still able to scroll down the list

**Additional context (optional)**
When adding a tag, people may have to scroll down the list to see more information

_I'm not sur if you can have any impact on this and it's just a nice to have_

**Comment by mhutchie** [comment]
Hi @rbalet,

Just to clarify, are you referring to the fact that whenever a dialog is open in the Git Graph View (the Add Tag dialog is just one example), you're unable to scroll the commits displayed in the Git Graph View?

**Comment by rbalet** [comment]
Hi @mhutchie.

Yes it s exactly that

**Comment by mhutchie** [comment]
Hi @rbalet,

I decided to prevent users from interacting with Git Graph View while a dialog is open, as there's the potential that their actions in the rest of the view may impact the dialog that was first opened (the underlying Git actions behind many dialogs impact each other). I also feel that creating a "read-only" mode, where the commits can be scrolled but not interacted with (e.g. opening context menu's), may cause unnecessary confusion.

As Visual Studio Code doesn't provide a feasible mechanism to create operating system native dialogs (as you would have in any other desktop application), this meant I had to implement the current dialogs as part of the Git Graph View. I've had to trade-off some capabilities of dialogs that would normally be built-in (e.g. allowing them to be dragged, allowing multiple to be open at the same time, etc.), with implementing Git capabilities requested by far more users of the extension. I haven't had any feedback from any other users that indicates the current dialogs are insufficient / not ideal, so it makes it hard to justify spending much time on overhauling the current dialogs. The feedback I've previously received is that users are quite happy to decide what they want to do before they open the appropriate dialog.

I'll close this request for now and move it to the backlog. However, if in the future more users express interest in enhancements to the dialogs (like this improvement request), I'll happily re-evaluate, re-open and implement it accordingly.
